src/lib/tableland-calls.ts

```diff
@@ -75,28 +91,45 @@ export async function create(
   tableId: string,
   options: CreateTableOptions
 ): Promise<CreateTableReceipt> {
-  const message = await GeneralizedRPC.call(
-    this,
-    "createTable",
-    query,
-    tableId,
-    options
-  );
+  const message = await GeneralizedRPC.call(this, "createTable", {
+    tableId: tableId,
+    statement: query,
+    dryrun: options.dryrun,
+    controller: true,
+  });
+
   const response = await SendCall.call(this, message);
   const json = await sendResponse(response);
 
-  return json;
+  return camelCaseKeys(json) as CreateTableReceipt;
```

Comment:
Is it possible to specify CreateTableReceipt from the result of sendResponse? That is a "safer" way if sendResponse is generic.